src/zkevm_specs/evm/execution/error_gas_uint_overflow.py

```diff
@@ -0,0 +1,108 @@
+from zkevm_specs.evm.util.call_gadget import CallGadget
+from ...util import (
+    FQ,
+    TxDataNonZeroGasEIP2028,
+    MAX_U64,
+    N_BYTES_U64,
+    TxGas,
+    TxGasContractCreation,
+    TxDataZeroGas,
+    # InitCodeWordGas,
+)
+from ..instruction import Instruction, Transition
+from ..table import CallContextFieldTag
+from ..opcode import Opcode
+
+
+def error_gas_uint_overflow(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    is_call = instruction.is_equal(opcode, Opcode.CALL)
+    is_create_flag = instruction.is_equal(opcode, Opcode.CREATE)
+    is_create2_flag = instruction.is_equal(opcode, Opcode.CREATE2)
```

Comment:
Seems is_delegatecall, is_callcode, is_staticcall need also being cover

```python
is_call, is_callcode, is_delegatecall, is_staticcall,  is_create, is_create2 = instruction.multiple_select(
        opcode, (Opcode.CALL, Opcode.CALLCODE, Opcode.DELEGATECALL, Opcode.STATICCALL, Opcode.CREATE, Opcode.CREATE2)
    ) 
    is_create = is_create_flag + is_create2_flag

    # Constrain opcodes.
    instruction.constrain_equal(is_call + is_callcode + is_delegatecall + is_staticcall + is_create + is_create2, FQ(1))
...    
call = CallGadget(instruction, FQ(0), is_call, is_callcode, is_delegatecall)

```

Reply:
I fixed!
I think that overflow happens not only CALL, CALLCODE, DELEGATECALL and STATICCALL, CREATE, CREATE2 so I don't add following constraint.
```py
instruction.constrain_equal(is_call + is_callcode + is_delegatecall + is_staticcall + is_create + is_create2, FQ(1))
```